components/admin_console/permission_schemes_settings/edit_post_time_limit_modal/index.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+import {ActionCreatorsMapObject, bindActionCreators, Dispatch} from 'redux';
+
+import {updateConfig} from 'mattermost-redux/actions/admin';
+import * as Selectors from 'mattermost-redux/selectors/entities/admin';
```

Comment:
maybe we should just import the one selector we are using here:

```suggestion
import {getConfig} from 'mattermost-redux/selectors/entities/admin';
```

Reply:
Done